x/btcstkconsumer/keeper/keeper.go

```diff
@@ -0,0 +1,59 @@
+package keeper
+
+import (
+	"fmt"
+
+	"cosmossdk.io/core/store"
+	"cosmossdk.io/log"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/babylonlabs-io/babylon/x/btcstkconsumer/types"
+)
+
+type Keeper struct {
+	cdc          codec.BinaryCodec
+	storeService store.KVStoreService
+
+	accountKeeper types.AccountKeeper
+	bankKeeper    types.BankKeeper
+	clientKeeper  types.ClientKeeper
+	wasmKeeper    types.WasmKeeper
```

Comment:
not being used
```suggestion
	clientKeeper  types.ClientKeeper
```

Reply:
Let's keep them for now as I see every module depends on accountKeeper / bankKeeper by default. Wdyt?

Reply:
In my opinion it only increases the complexity if it is not being used, but it is your call on this ^^

Reply:
this is a bit artefact of the past i.e first Babylon modules we created, we copied them from ignite tool (from what I remember) and ignite by default added those keepers, so we kinda rolled with it as we did not know any better back then.

We should clean this up, and every keeper should only depend on what is needed. Maybe separate task ?